app/views/component-detail.html

```diff
@@ -12,7 +12,11 @@
 				{% if has_css %}<span class="label-css">CSS</span>{% endif %}
 				{% if has_js %}<span class="label-js">JS</span>{% endif %}
 
-				<p>{{description}}</p>
+				{% if design_guidelines %}
+					{{design_guidelines|markdown|raw}}
+				{% else %}
+					<p>{{description}}</p>
```

Comment:
Let's rename the description field to "strapline" as we'll still want it to appear on the search index but it will serve a different purpose to the design_guidelines